website/docs/r/organizations_account.html.markdown

```diff
@@ -12,8 +12,6 @@ Provides a resource to create a member account in the current organization.
 
 ~> **Note:** Account management must be done from the organization's master account.
 
-!> **WARNING:** Deleting this Terraform resource will only remove an AWS account from an organization. Terraform will not close the account. The member account must be prepared to be a standalone account beforehand. See the [AWS Organizations documentation](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_remove.html) for more information.
```

Comment:
I would _reword_ this warning to make it clear that you should think about the behavior you want when someone deletes the resource.